lib/widgets/followers.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+
+Row followers() {
```

Comment:
Here also too, either let the function stay in the same screen class or separate it with standalone widget.